addons/website/models/ir_ui_view.py

```diff
@@ -13,12 +13,10 @@
 
 
 class view(osv.osv):
-    _inherit = "ir.ui.view"
+    _name = "ir.ui.view"
+    _inherit = ["ir.ui.view", "website.seo.metadata"]
```

Comment:
Metadata related field are also available in website.seo.metadata model so remove fields from ir.ui.view and inherit website.seo.metadata.
OK. i have put in separate commit